internal/provider/kubernetes/kubernetes_test.go

```diff
@@ -468,6 +468,14 @@ func testHTTPRoute(ctx context.Context, t *testing.T, provider *Provider, resour
 		require.NoError(t, cli.Delete(ctx, authenFilter))
 	}()
 
+	rateLimitFilter := test.GetRateLimitFilter("test-ratelimit", ns.Name)
+
+	require.NoError(t, cli.Create(ctx, rateLimitFilter))
+
+	defer func() {
+		require.NoError(t, cli.Delete(ctx, rateLimitFilter))
+	}()
+
```

Comment:
Can you create an HTTPRoute that references the RateLimitFilter and ensure the resource map contains the expected rate limit? See the following for references:

https://github.com/envoyproxy/gateway/pull/791/files#diff-5f29f313b6f74b9836b294d6fe03a909e7e023a42655873eb1a13b316e624e7d

https://github.com/envoyproxy/gateway/pull/813/files#diff-5f29f313b6f74b9836b294d6fe03a909e7e023a42655873eb1a13b316e624e7d